ethereum/spec/src/main/java/tech/pegasys/teku/spec/schemas/SchemaDefinitionsAltair.java

```diff
@@ -87,6 +88,16 @@ public BeaconBlockSchema getBeaconBlockSchema() {
     return beaconBlockBodySchema;
   }
 
+  @Override
+  public BeaconBlockSchema getBlindedBlockSchema() {
+    return getBeaconBlockSchema();
+  }
+
+  @Override
+  public BeaconBlockBodySchema<?> getBlindedBlockBodySchema() {
+    return getBeaconBlockBodySchema();
+  }
```

Comment:
@tbenr would be interested if you think this is enough?

I'm not sure it'd add anything but noise adding all the blinded interfaces, but would welcome thoughts...

Reply:
It's borderline to me. The underlying question would be: do we want to support blinded api calls in non-Bellatrix milestones? The answer seems to be yes, so for the moment I think we can leave it there